### PR TITLE
Expand the README with instructions for contributing & building

### PR DESCRIPTION
**Pull request by emackey**
This should offer more guidance to those seeking to make corrections.

New version: [README.md](https://github.com/KhronosGroup/glTF/blob/bec27241a2082bbec2ae4f884357d19a2cfa91c8/specification/2.0/README.md)

/cc @javagl @lexaknyazev

### CODE REVIEW

**Comment by javagl** [comment]
Although I haven't tried out the build process (and I see that as a TODO for me), the instructions here will certainly be helpful, and the clarification in the 'Contributing' section about the auto-generated parts will be a useful guideline for contributors, 👍

File: specification/2.0/README.md [hunk 1]
```diff
@@ -3,13 +3,56 @@ Copyright 2014-2021 The Khronos Group Inc.
...
+The full specification is available in two forms:
+
+- [Specification as HTML](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html) - for viewing in a browser
+- [Specification as PDF](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.pdf) - for viewing as a document
+
+## Extensions
+
+Features not covered by the core glTF specification may be contributed by way of extensions. See the [glTF Extension Registry](https://github.com/KhronosGroup/glTF/tree/main/extensions) for details.
+
+## Contributing
```
Comment (lexaknyazev) [inline specification/2.0/README.md:19]: There's also a repo-level [CONTRIBUTING](https://github.com/KhronosGroup/glTF/blob/main/CONTRIBUTING.adoc) file that is being linked under each Issue and PR form. Should it be updated with a link to this section?

File: specification/2.0/README.md [hunk 2]
```diff
@@ -3,13 +3,56 @@ Copyright 2014-2021 The Khronos Group Inc.
...
+```
+docker pull khronosgroup/docker-images:asciidoctor-spec
+```
+
+Then, open a terminal inside the new container, and do the following:
+
+1. `git clone ...` this repository, or your fork.
+2. `git checkout ...` the branch with your changes.
+3. `cd specification/2.0`
+4. `make Specification.html Specification.pdf`
```
Comment (lexaknyazev) [inline specification/2.0/README.md:52]: Just use `make` to not depend on temporary filenames.

Comment (emackey) [reply specification/2.0/README.md:52]: Last I checked, our current Makefile doesn't support leaving out these filenames.  I could check again (later, from another machine...)

Comment (emackey) [reply specification/2.0/README.md:52]: You're right that some lines at the top make it look like this should have worked for me when I tried it.  Maybe my build targets were already up-to-date at the time, I forget.

File: specification/2.0/README.md (hunk 1 above)
Comment (emackey) [reply specification/2.0/README.md:19]: Added.

File: specification/2.0/README.md (hunk 2 above)
Comment (emackey) [reply specification/2.0/README.md]: Yep, not sure how I messed this up earlier, but `make` is indeed working fine.  Corrected.

File: specification/2.0/README.md [hunk 3]
```diff
@@ -3,13 +3,56 @@ Copyright 2014-2021 The Khronos Group Inc.
...
+
+- **Section 5 (Properties Reference)** - This section is auto-generated using the [JSON Schema files](https://github.com/KhronosGroup/glTF/tree/main/specification/2.0/schema). Corrections to the wording must be made directly in those JSON files. Changes to the JSON structure are not being accepted at this time.
+
+- **Appendix A (JSON Schema Reference)** - This section is likewise auto-generated using the [JSON Schema files](https://github.com/KhronosGroup/glTF/tree/main/specification/2.0/schema).
+
+- **All other sections** - The remainder of the specification is contained in [Specification.adoc](https://github.com/KhronosGroup/glTF/blob/main/specification/2.0/Specification.adoc). General edits may be made here.
+
+## Building with GitHub Actions
+
+Opening a PR here will trigger a GitHub Action. Click on the "Actions" tab of the repository containing the edited branch, making careful note that this is likely to be your own fork of the glTF repository, not the main one. The most recent Action should display the name of your commit or PR.
```
Comment (lexaknyazev) [inline specification/2.0/README.md:33]: To avoid potential confusion with the branch name, "not the main one" -> "not the Khronos one".

Comment (emackey) [reply specification/2.0/README.md]: Done.

**Review by lexaknyazev — APPROVED** [review]

**Pull request merged**